docs/specs/README.md

```diff
@@ -0,0 +1,64 @@
+# Guidelines for Feature Request(s)
+
+This document covers, how to contribute new feature(s) or significant design changes to BigchainDB. Since, contribution
+of such functionality involves design discussion and might take multiple releases to complete,
+we expect the contributors to follow certain guidelines before proceeding with the implementation.
+
+
+## Classification
+How to classify your contribution as a feature and determine if it needs design discussion. 
+
+Here are some baseline criterion that can help you classify your idea, as a feature:
+
+- Requires significant effort/changes to the current BigchainDB implementation and it is not self
+ contained (e.g. API changes,  introducing a new component, takes more than 20 Wrigleys
+ (Wrigley is a `man day` @ BigchainDB))
+- Impacts the clients and drivers.
+- Can we write a blog post about it?
+- Requires collaboration with multiple teams in terms of development, operations and testing.
+- Impacts documentation significantly.
+
+It is not a feature if:
+
+- It is straightforward enough and might not need a design discussion.
+  - Test case fixes
+  - Fixing typos/grammar in documentation.
+  - Refactoring code
+  - Adding error or log messages.
+
+## Classified as a Feature?
+Create a new issue on our [GitHub repository](https://github.com/bigchaindb/bigchaindb/issues/new) with a
+`Feature-Request` label, once you have classified your proposal as a feature and:
```

Comment:
I like the label! Maybe there could also be a 'discussion needed' label. So people can raise attention to an issue which is still missing some information/some decisions have to be made. That's actually something I thought we could add in general.

Reply:
I already have a label `Spec`, shouldn't that be sufficient? But I can add that in the document as well.

Reply:
I guess in this case it might be enough.